src/Modules/Shared/PowerShellGet/PSModule.psm1

```diff
@@ -13008,8 +13008,8 @@ function Update-ModuleManifest
     {
         #DscResourcesToExport field is not available in PowerShell version lower than 5.0
         
-        if  (($PSVersionTable.PSVersion -lt [Version]"5.0") -or ($PowerShellVersion -and $PowerShellVersion -lt [Version]"5.0") `
-             -or (-not $PowerShellVersion -and $moduleInfo.PowerShellVersion -and $moduleInfo.PowerShellVersion -lt [Version]"5.0") `
+        if  (($PSVersionTable.PSVersion -lt '5.0.0') -or ($PowerShellVersion -and $PowerShellVersion -lt '5.0') `

```

Comment:
@quoctruong, not required for now, $PowerShellVersion is still of Version type and not SemanticVersion. We will have to make lot of changes to support SemanticVersion type in the module infrastructure.